code/datums/spells/horsemask.dm

```diff
@@ -26,7 +26,8 @@
 	if(!target)
 		return
 
-	if((target.type in compatible_mobs) || ishuman(target))
+
+	if(!is_type_in_list(target, compatible_mobs))

```

Comment:
Since this is only ever used on human targets, why not just check for `!ishuman(target)` and get rid of the compatible mobs list?

The only reason this is a list is because on TG monkies are a carbon. 

Doubt we'll ever get a new mob that can have clothing equipped to it in the same manner as humans.